db-migration/src/lib.rs

```diff
@@ -46,15 +52,24 @@ impl Migrations {
             None => {
                 // if version is none, but db is not empty
                 // patch 220464f
-                return self.is_non_empty_rdb(db);
+                if self.is_non_empty_rdb(db) {
+                    return Ordering::Less;
+                } else {
+                    return Ordering::Equal;
+                }
             }
         };
+        debug!("current database version [{}]", db_version);
 
-        self.migrations
+        let latest_version = self
+            .migrations
             .values()
             .last()
-            .map(|m| m.version() > db_version.as_str())
-            .unwrap_or(false)
+            .unwrap_or_else(|| panic!("should have at least one version"))
+            .version();
+        debug!("latest  database version [{}]", latest_version);
```

Comment:
```suggestion
        debug!("latest database version [{}]", latest_version);
```

Reply:
Oh, this is not typo, I want to align the version number with the above debug message:
````
debug!("current database version [{}]", db_version);
debug!("latest  database version [{}]", latest_version);
````